src/IISLib/util.cxx

```diff
@@ -61,13 +61,17 @@ Return Values:
             return hr;
         }
     }
-    else
+    else if (wcslen(pszName) > MAX_PATH)
```

Comment:
I feel like it would be better to check this in https://github.com/aspnet/IISIntegration/blob/18799f6d4847dbad82efce25657cada221fa309f/src/CommonLib/utility.cxx#L427.
Check:
```c++
if (wcslen(pszFullPath) > MAX_PATH)
{
    hr = MakePathCanonicalizationProof( pszFullPath, pStruFullPath );
}
```

Reply:
I don't want to touch network path (`\\share\bla`) logic at all and that would affect it.

Reply:
Kk. @pan-wang this will be modifying IISLib code.